versioned_docs/version-1.3.0-beta/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,178 @@
+}
+```
+## Download your platform specific artifact
+
+Pull the artifact to that directory:
+```console
+$ oras pull localhost:15000/oras:v1 --platform linux/amd64 -o ./download
+✓ Skipped     application/vnd.unknown.config.v1+json                                                                                    37/37  B 100.00%     0s
+  └─ sha256:9d99a75171aea000c711b34c0e5e3f28d3d537dd99d110eafbfbc2bd8e52c2bf
+✓ Pulled      bin/linux/amd64/oras                                                                                                      11/11 MB 100.00%   19ms
+  └─ sha256:4640fefdb4e22eb1308e40d5fac9fd9d16e17dfbc20202bb3ed42033d01d823e
+✓ Pulled      application/vnd.oci.image.manifest.v1+json                                                                              581/581  B 100.00%   81µs
+  └─ sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+Skipped pulling layers without file name in "org.opencontainers.image.title"
+Use 'oras copy localhost:15000/oras:v1 --to-oci-layout <layout-dir>' to pull all layers.
+```
+
+Validate your download
+```console
+$ file download/bin/linux/amd64/oras
+download/bin/linux/amd64/oras: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=g5htPJf4jOBYObk8HPGK/C_STgrrtpidMtnOXIshh/AwKGiCqqgvmy7kvjXixL/hvqsAg3dicj7a6T6waJe, not stripped
+$ sha256sum ./download/bin/linux/amd64/oras
+4640fefdb4e22eb1308e40d5fac9fd9d16e17dfbc20202bb3ed42033d01d823e  ./download/bin/linux/amd64/oras
+```
+
+## Update your multi-architecture manifest
+
+You can also update your manifest if you want to add a new artifact.
+For example, say you created an executable for Darwin, push the executable:
+```console
+$ oras push localhost:15000/oras:darwin,amd64  --artifact-platform darwin/amd64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/darwin/amd64/oras:application/octet-stream
+✓ Exists    bin/darwin/amd64/oras                                                                                                    11.1/11.1 MB 100.00%     0s
+  └─ sha256:a839f5d9cd3502b87b8d8edca31cd9f6f66c4835364e73b87a1645353631590f
+✓ Exists    application/vnd.unknown.config.v1+json                                                                                       38/38  B 100.00%     0s
+  └─ sha256:e9302bbb2fb8f6c2df866d3c4e41917849442f89a575f36f43366a7279804f70
+✓ Uploaded  application/vnd.oci.image.manifest.v1+json                                                                                 589/589  B 100.00%     0s
+  └─ sha256:f5758b75ebc4d82280078afb9ebeb859f91a481667c018c9edf91a432518cb20
+Pushed [registry] localhost:15000/oras:darwin
+Tagged amd64
+ArtifactType: application/vnd.oci.image.config.v1+json
+Digest: sha256:f5758b75ebc4d82280078afb9ebeb859f91a481667c018c9edf91a432518cb20
+````
+
+Add it to your manifest:
+```console
+$ oras manifest index update localhost:15000/oras:v1 --add sha256:f5758b75ebc4d82280078afb9ebeb859f91a481667c018c9edf91a432518cb20
```

Comment:
nit: Should we define an environment variable to reference the digest for `darwin/amd64` like we did in the above examples?